pkg/apiserver/middlewares/v1/jwt.go

```diff
@@ -147,14 +147,16 @@ func NewJWT(dbClient *database.Client) (*JWT, error) {
 		secret []byte
 	)
 
+	//Please be aware that brute force HS256 is possible.
+	//PLEASE choose a STRONG secret
 	secret_string := os.Getenv("CS_LAPI_SECRET")
 
 	if secret_string == "" {
-		secret = make([]byte, 8)
-		if n, err := rand.Reader.Read(secret); err != nil {
+		secret = make([]byte, 64)
+		if n, err := rand.Read(secret); err != nil {
```

Comment:
is 64 arbitrarily chosen or is there any kind of best practice we can refer to ?

Reply:
It's the length used at auth0.